src/SprykerSdk/Spryk/Model/Spryk/Configuration/Validator/Rules/LevelRule.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Configuration\Validator\Rules;
+
+class LevelRule implements ConfigurationValidatorRuleInterface
+{
+    protected const LEVEL_CONFIG_NAME = 'level';
+    /**
```

Comment:
Please add empty string after the constant.

Reply:
done